docs/getting-started/index.md

```diff
@@ -1,20 +1,18 @@
 ---
 layout: default
-title: Getting-started
+title: Getting Started
 permalink: /getting-started
 nav_order: 3
 has_children: true
 has_toc: true
 ---
 
-# Getting started
+# Getting Started
 
 GrimoireLab is a set of tools, and to ease the trouble with installation we are
 providing a default setup to analyze Git activity for the grimoirelab
 repository. In order to have this default setup running, follow the [setup
-grimoirelab](/docs/getting-started/setup)
-section.
+grimoirelab](docs/getting-started/setup) section.
```

Comment:
Sometimes having path works but sometimes it doesn't. Should every redirecting be converted to the baseurl stuff ? 
WDYT ?